docs/source/pillars/computing_technology.md

```diff
@@ -2,55 +2,56 @@
 
 # Computing technology
 
-This capability concerns what the TRE operator does to manage systems for storing, retrieving, analysing and sending information.
-
 ```{figure} ../../images/Capability_Map/full.drawio.svg
 :alt: SATRE Pillars Capability Map
 :align: center
 
 SATRE Pillars Capability Map
 ```
 
+This pillar concerns actions taken by the TRE operator to manage TRE computing systems.
+
+Each TRE operator will have its own computing technology requirements.
+The security controls needed by the computing infrastructure will depend on information governance requirements.
+Other computing requirements will be influenced by the technical knowledge and experience of those using the TRE, along with the work they need to perform within the system. 
+For example, a data scientist will have very different requirements to a clinician.
+The required compute resources will vary according to the scale of data and computational techniques employed during research.
+
 <!-- See all pillars of the SATRE Pillars Capability Map here: {ref}`satre_pillars` -->
 
 ## End user computing
 
-_The ability of the TRE operator to provide and manage devices, workspaces, interfaces and applications used by researchers to interact with underlying systems and data._
+The ability of the TRE operator to provide and manage devices, workspaces, interfaces and applications used by researchers to interact with underlying systems and data.
 
-### User interface
+### End user computing interfaces
 
-_The interfaces used for interacting with the TRE management system and the TRE workspace._
+Software or systems that allows people to interact with the TRE.
+This may include desktop, command-line and/or code-submission interfaces.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-end-user-user-interface
+:name: tab-end-user-computing-interfaces
 
 * - Statement
   - Guidance
   - Importance
-* - Your TRE should be accessed via a user interface accessible using commonly available applications.
-  - TREs which allow users to connect from their own devices should not require the installation of any bespoke TRE application on the user's device.
-    In practice a web browser is the most common way to achieve this.
+
+* - You should disable the ability to copy data out of your TRE via the system clipboard.
+  - A TRE user must not be able to copy sensitive data out of a workspace using the system clipboard.
+    A TRE may allow user to paste text into a workspace.
   - Recommended
```

Comment:
```suggestion
* - You must not allow users to copy data out of your TRE via the system clipboard.
  - A TRE user must not be able to copy sensitive data out of a workspace using the system clipboard.
    A TRE may allow user to paste text into a workspace.
    This might not be relevant to your TRE, for example if your user interface does not have a clipboard.
  - Required
```